pipelines/build/regeneration/jdk11_regeneration_pipeline.groovy

```diff
@@ -20,7 +20,20 @@ node ("master") {
   try {
     def scmVars = checkout scm
     load "${WORKSPACE}/pipelines/build/common/import_lib.groovy"
-  
+
+    // Load gitUri and gitBranch. These determine where we will be pulling configs from.
+    def repoUri = "$REPOSITORY_URL" != "" ? REPOSITORY_URL : "https://github.com/AdoptOpenJDK/openjdk-build.git"
+    def repoBranch = "$REPOSITORY_BRANCH" != "" ? REPOSITORY_BRANCH : "master"
```

Comment:
These params end up being forwarded on and being used for the job config for all the build jobs (eg jdk11u-mac-x64-openj9). That means the entire build is now running from my repo/branch where I may have made a few changes to jdk11u.groovy for example. I am now forced to keep this entire branch up to date with the latest from the main repo so I don't fall behind. Which means I need to maintain a mirror job to do an automated merge. My ideal scenario is that I can keep a separate repo with a few config files which are a modification of the default ones. This repo can then feed in to the builds and generators. The only time my repo breaks is when the definition of that config file (api if you will) changes. I no longer need to worry about keeping a repo in sync. I _should_ ensure my few files are in sync, but if they are defined properly, I shouldn't break when things change, I should fall back to defaults, possibly with a warning that I am missing some lines in the configs.

This is how we do it with the OpenJ9 build scripts
https://github.com/eclipse/openj9/blob/master/buildenv/jenkins/variables/defaults.yml
Everything we've ever needed to change between the farms is defined in this file and if anyone wants to override any of them on demand you can do so via param. It's convenient to have everything in one place.

I'm not saying either way is right or wrong, but this is what I know and where my bias lies. I am always open for doing things better. It just seems that in its current state, this PR has not addressed all the requirements I hoped for.

I will leave it at that for now. I have reviewed most of this but I think I will pause for now so we can have some more discussion. If it's more efficient, we can book a call with some of the stakeholders.